### PR TITLE
Declare keepass gem only for password generation

### PR DESCRIPTION
**Pull request by kamaradclimber**
It is really useless to install it otherwise.

### CODE REVIEW

File: recipes/local_user.rb [hunk 1]
```diff
@@ -19,12 +19,13 @@
...
 
-chef_gem 'keepass-password-generator' do
-  version '0.1.1'
-end
-
 user_pwd = node['rundeck_node']['user_password']
 if node['rundeck_node']['user_password_file']
+
+  chef_gem 'keepass-password-generator' do
+    version '0.1.1'
```
Comment (brugidou) [inline recipes/local_user.rb]: why do you specify version

Comment (kamaradclimber) [inline recipes/local_user.rb]: to keep all nodes consistent

Comment (Annih) [inline recipes/local_user.rb]: maybe use an attribute here in order to keep consistency and flexibility ?

**Comment by Annih** [comment]
:+1:  let's merge :)

**Pull request merged**